src/sources/audio_file/mod.rs

```diff
@@ -133,17 +133,17 @@ impl Default for AudioFileSettings {
 
 fn play_sound_error_transmute<Out>(err: PlaySoundError<()>) -> PlaySoundError<Out> {
     match err {
-        PlaySoundError::CommandError(cmd) => PlaySoundError::CommandError(cmd),
         PlaySoundError::SoundLimitReached => PlaySoundError::SoundLimitReached,
-        _ => unreachable!(),
+        _ => unreachable!(
+            "Cannot magically go from () to {}",
```

Comment:
Maybe something like this to be slightly more professional 😛 
```suggestion
            "Cannot convert from () to {}",
```

Reply:
lol I don't remember writing it like that, you're right that it's a much better message 😆